demo/customhtml.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  html: `
```

Comment:
Should we make this a function that receives the config? That way, people can inject the test framework specified in the config.

Reply:
Yeah was contemplating something like that as well. Will update later 🙂

Reply:
ptal